guides/moving-to-oss.md

```diff
@@ -0,0 +1,235 @@
+
+|                                         | Coder v1                                                                                                             | Coder OSS                                                                                                                                                             |
+| --------------------------------------- | -------------------------------------------------------------------------------------------------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Kubernetes                              | ✅ Helm chart                                                                                                        | ⌛ Helm chart [(needs docs)](https://github.com/coder/coder/issues/3224)                                                                                              |
+| Kubernetes (HA/multiple replicas)       | ✅                                                                                                                   | ⌛ [#3502](https://github.com/coder/coder/issues/3502)                                                                                                                |
+| Kubernetes (built-in database)          | ✅                                                                                                                   | ❌                                                                                                                                                                    |
+| Docker deployment                       | ✅                                                                                                                   | ✅                                                                                                                                                                    |
+| VM deployment                           | ❌                                                                                                                   | ✅ [system packages](https://coder.com/docs/coder-oss/latest/install#system-packages) and [prebuilt binaries](https://coder.com/docs/coder-oss/latest/install#manual) |
+| Built-in PostgreSQL                     | ✅                                                                                                                   | ✅                                                                                                                                                                    |
+| Built-in TLS tunnel                     | ❌                                                                                                                   | ✅                                                                                                                                                                    |
```

Comment:
This is going to get out of hand if you plan to list every feature that OSS has but v1 does not, given that we are winding down feature work on v1 and ramping up on OSS.

This doc could really use some culling to just the most important differences.

Reply:
Makes sense. I removed the tunnel. I think it's important to list most v1 features, but not necessarily most OSS features.